api/all/src/main/java/io/opentelemetry/api/DefaultOpenTelemetry.java

```diff
@@ -46,4 +46,12 @@ public MeterProvider getMeterProvider() {
   public ContextPropagators getPropagators() {
     return propagators;
   }
+
+  @Override
+  public String toString() {
+    return "DefaultOpenTelemetry{"
```

Comment:
Still trying to figure out how that is correct. TracerProvider and MeterProvider are always noop. Should I include them in `toString`?

Reply:
I'd say no. This looks fine to me.